python/ray/autoscaler/aws/example-cloudwatch.yaml

```diff
@@ -0,0 +1,86 @@
+            # 4. log stream name will be the head node instance id in this example.
+            # See https://docs.ray.io/en/master/ray-logging.html for ray logging system details.
+
+            # This tag serve as an unique identifier to AMI name with Unified CloudWatch Agent installed.
+            CLOUDWATCH_AGENT_INSTALLED_AMI_TAG: "T6Iq2faj"
+            config: "cloudwatch/example-cloudwatch-agent-config.json"
+            retryer:
+                # Max allowed Unified CloudWatch Agent installation attempts on any host.
+                max_attempts: 120
+                # Seconds to wait between each Unified CloudWatch Agent installation attempt.
+                delay_seconds: 30
+
+# How Ray will authenticate with newly launched nodes.
+auth:
+    ssh_user: ubuntu
+
+available_node_types:
+  ray.head.default:
+      node_config:
+        InstanceType: c5a.large
+        # Disclaimer: CloudWatch integration with Ray requires an AMI (or Docker image) with the Unified CloudWatch Agent pre-installed.
+        # The AMI below is provided by the Amazon Ray Team, is based on version 48 of the Ubuntu 18.04 AWS Deep Learning AMI,
+        # and ships with Unified CloudWatch Agent v1.247348.0b251302.
+        # Please direct any questions, comments, or issues to the Amazon Ray Team at https://github.com/amzn/amazon-ray/issues/new/choose.
+        # Up-to-date versions of this AMI and AMIs for other regions can be found at https://github.com/amzn/amazon-ray.
+        ImageId: ami-01e67c19a261661bb
+      resources: {}
+      max_workers: 10
+  ray.worker.default:
+      node_config:
+        InstanceType: c5a.large
+        ImageId: ami-01e67c19a261661bb
+        # Note: IamInstanceProfile is needed to grant worker nodes required permission to make boto3 call for Cloudwatch setup.
+        # Default IamInstanceProfile is arn:aws:iam::{your_aws_account_number}:instance-profile/ray-autoscaler-cloudwatch-v1.
+        IamInstanceProfile:
+          Name: ray-autoscaler-cloudwatch-v1
+      resources: {}
+      min_workers: 1
+head_node_type: ray.head.default
+
+# Custom commands that will be run on the head node after common setup.
+# Uncomment below if you want to export Ray system-level Prometheus metrics to Cloudwatch.
+# ray_prometheus_waiter.sh is a bash script to wait for ray Prometheus service discovery file at /tmp/ray/prom_metrics_service_discovery.json to appear then restart cloudwatch agent to capture Prometheus metrics.
+# See https://docs.ray.io/en/latest/ray-metrics.html for details about exporting ray Prometheus metrics.
+# prometheus.yml is the configuration file defining scraping jobs. Here it is configured to scrape all ray metrics.
+# See https://prometheus.io/docs/prometheus/latest/configuration/configuration/ for details about Prometheus configuration file.
+# Note: Prometheus to Cloudwatch support requires Unified CloudWatch Agent installed and ray dashboard support.
```

Comment:
Should these be commented in this example? 

If someone wanted to just try this out, I assume they would need to un-comment them?